Cloud Application Manager/Tutorials/ebcli-tutorial.md

```diff
@@ -0,0 +1,442 @@
+{{{
+"title": "Using ebcli",
+"date": "05-20-2019",
+"author": "Diego Sanjuan",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+**In this article:**
+
+* [Overview](#overview)
+* [Audience](#audience)
+* [Prerequisites](#prerequisites)
+* [Installation of ebcli command line tool](#installation-of-ebcli-command-line-tool)
+* [Login into your Cloud Application Manager account with ebcli](#login-into-your-cloud-application-manager-account-with-ebcli)
+* [Show available ebcli commands](#show-available-ebcli-commands)
+* [Box commands using ebcli](#box-commands-using-ebcli)
+* [Instance lifecycle using ebcli](#instance-lifecycle-using-ebcli)
+* [ECS Image management using ebcli](#ecs-image-management-using-ebcli)
+* [Workspaces list using ebcli](#workspaces-list-using-ebcli)
+* [Delete Resources using ebcli](#delete-resources-using-ebcli)
+* [Logout from your Cloud Application Manager account with ebcli](#logout-from-your-cloud-application-manager-account-with-ebcli)
+* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+
+
+### Overview
+
+This tutorial is meant to assist users of Cloud Application Manager to learn how to use [ebcli](https://pypi.python.org/pypi/ebcli) - our command line tool- to control their Cloud Application Manager account as follows:
+
+* For boxes - how to deploy, export, import as version or as a draft, etc using ebcli.
+* For instances - how to reconfigure, reinstall, shutdown, poweron, export, etc using ebcli.
+* For ECS images - how to manage them using ebcli.
+* For workspaces - how to list them using ebcli.
+
+ebcli calls Cloud Application Manager APIs in the backend to list, provision and manage the lifecycle of your workloads based on the box configuration.
+
+### Audience
+
+All Cloud Application Manager users who want to control their Cloud Application Manager account using ebcli - command line tool.
+
+
+### Prerequisites
+
+* Access to Cloud Application Manager [Site](https://cam.ctl.io/#/login) or to an on-premises CAM appliance.
+* The user must have an existing account to login into using username and password or his preferred authentication mechanism.
+
+
+## Installation of ebcli command line tool
+
+Before you start using ebcli, you need to install it using pip which is a package manager for Python packages, or modules if you like.
+
+If you don't have pip installed. Please follow this guide to install it into your computer: https://pip.pypa.io/en/stable/installing/
```

Comment:
```suggestion
If you don't have `pip` installed. Please follow this guide to install it into your computer: https://pip.pypa.io/en/stable/installing/
```
Please, try to include any reference to commands or code in a preformatted block